pkg/fedora/tarsnap.spec

```diff
@@ -0,0 +1,59 @@
+Name:           tarsnap
+Version:        1.0.36.1
+Release:        2%{?dist}
+Summary:        Secure, efficient online backup service
+
+License:        Proprietary
+URL:            http://www.tarsnap.com/
+Source0:        http://www.tarsnap.com/download/tarsnap-autoconf-%{version}.tgz
+
+BuildRequires:  gcc
+BuildRequires:  pkgconfig(bash-completion)
+BuildRequires:  pkgconfig(bzip2)
```

Comment:
See below as to the bzip2 dependency.  `BuildRequires: pkgconfig(foo)` [is preferred](https://fedoraproject.org/wiki/Packaging:PkgConfigBuildRequires) to `BuildRequires: foo-devel` for dependencies that are detected via pkg-config, since the former is more robust to package renames.  The CentOS 7 version of bzip2-devel doesn't have a pkg-config file, so this fails.

In any event, I misremembered the packaging guideline; it does *not* apply to packages detected by hand, even if they ship with a pkg-config file.  So, that notation should only be used for `bash-completion`.  I'll update the specfile.